beast/fitting/trim_grid.py

```diff
@@ -29,33 +29,25 @@ def trim_models(
     ----------
     sedgrid: grid.SEDgrid instance
         model grid
-
     sedgrid_noisemodel: beast noisemodel instance
         noise model data
-
     obsdata: Observation object instance
         observation catalog
-
     sed_outname: str
         name for output sed file
-
     noisemodel_outname: str
         name for output noisemodel file
-
-    sigma_fac: float
+    sigma_fac: float, optional
```

Comment:
want a space between parameter name and : to be consistent with your updates above?

Reply:
Yep.